hash.c

```diff
@@ -92,23 +92,27 @@ void *hashFind(const hash_t *pHash, const janus_id_t id) {
   return pResult;
 }
 
-switch_status_t hashDelete(const hash_t *pHash, const janus_id_t id) {
+switch_status_t hashDeleteStr(const hash_t *pHash, char *pIdStr) {
```

Comment:
In hash header, it supposed to be have this two function
- hashDelete
- hashDeleteStr

but this change is hashDeleteStr overwrites the hashDelete.

Reply:
i think it is the artifact of the unified diff, both functions are there